oaknational/oak-components
# PUPIL-180 Feat/update tokens

update tokens to match design kit as documented here
https://docs.google.com/spreadsheets/d/1yZHdogEudP9TPu2vEthy3OM3Ghr50ef-w7RYBofz2iQ/edit?usp=sharing

### to test

check the storybook - specifically the StyleTokens section
https://deploy-preview-37--lively-meringue-8ebd43.netlify.app/
